docs/admin/maintenance/noble_migration_prep.rst

```diff
@@ -0,0 +1,134 @@
+Preparing for the Ubuntu 24.04 (Noble) migration
+================================================
+
+In 2025, SecureDrops will need to be upgraded to the newer Ubuntu 24.04 (Noble)
+operating system. This process will be more straightforward than in the past
+as there will be semi-automated and fully automated upgrade processes.
+
+At this time, the current focus is on ensuring all SecureDrop servers are in a good state
+to be ready for the migration. SecureDrop will automatically check some conditions, and report
+via an alert in the Journalist Interface and OSSEC alerts if there are issues.
+
+Timeline
+--------
+
+You should ensure all errors are resolved by January 31, 2025, to ensure your SecureDrop
+servers can safely proceed to the next stage of the migration.
+
+Getting more details
+--------------------
+
+If you see the alert in the Journalist Interface or receive an OSSEC alert, you'll
+need to :doc:`SSH to the Application and Monitor Servers <../installation/test_the_installation>`
+to get more information on what specifically is failing.
+
+Once logged in, run:
+
+.. code:: sh
+
+   sudo securedrop-noble-migration-check
+
+It will display a number of checks and whether they are failing.
+Steps to address each issue are listed below. If you are unsure what to do,
+please :ref:`contact Support <getting_support>`. It is safe to run this command
+multiple times, e.g. if you resolved an issue and want to see that it is fixed.
+
+For example:
+
+.. code:: sh
+
+    $ sudo securedrop-noble-migration-check
+    ssh: group is empty
+    ufw ERROR: ufw is still installed
+    free space: not enough free space, have 88994088, need 4305364600
+    apt: all sources are expected
+    systemd: no failed units
+
+    Some errors were found that will block migration.
+
+    Documentation on how to resolve these errors can be found at:
+    <https://docs.securedrop.org/en/stable/admin/maintenance/noble_migration_prep.html>
+
+    If you are unsure what to do, please contact the SecureDrop
+    support team: <https://docs.securedrop.org/en/stable/getting_support.html>.
+
+In this case the "ufw" check failed.
```

Comment:
While true, I think this might be confusing - The "ufw" check failed, but in that output it also failed the SSH group and free space checks, too.

I think omitting that line completely is fine, since we dive directly into the explanations below.

Reply:
Oops, I meant to edit the free space check. But in this case the SSH group did pass; the group is supposed to be empty. (That's why I want to add the "ERROR" labels, to make it clear.)

Reply:
Maybe...`the "ERROR" indicates that only the "ufw" check failed`?

Reply:
Done, and fixed the free space check. Let me know what you think.